content/blog/nats-server-29-release.md

```diff
@@ -0,0 +1,489 @@
+
+This release brings a handful improvements including a new [encryption][encryption] choice for JetStream file storage, a new account purge operation, new subject mapping functions for more powerful transforms, and the ability to define user templates for scoped signing keys.
+
+[security]: https://docs.nats.io/nats-concepts/security
+[encryption]: https://docs.nats.io/running-a-nats-service/nats_admin/jetstream_admin/encryption_at_rest
+
+#### AES-GCM cipher for JetStream file-based encryption
+
+JetStream has had support for [encryption at rest][encryption] since 2.3.0, specifically using the [ChaCha20-Poly1305][chacha] algorithm. This release adds support for [AES-GCM][aes-gcm], which may be desired or required for some organizations.
+
+To enable encryption, set the key and the cipher in the `jetstream` block of the server configuration.
+
+```
+jetstream {
+  key: $JETSTREAM_KEY
+  cipher: aes
+}
+```
+
+*Note: as a best practice, the key should *not* be inlined in the configuration file. Instead, it can expressed as [variable][variable] using the `$` notation which will get interpolated from the environment by default.*
+
+[encryption]: https://docs.nats.io/running-a-nats-service/nats_admin/jetstream_admin/encryption_at_rest
+[chacha]: https://pkg.go.dev/golang.org/x/crypto/chacha20poly1305
+[aes-gcm]: https://pkg.go.dev/crypto/aes
+[variable]: https://docs.nats.io/running-a-nats-service/configuration#variables
+
+#### Account purge operation
+
+For operators managing accounts, it may be desired to purge all resources associated with an account. A new API available to the system account has been implemented to support this:
+
+```
+$JS.API.ACCOUNT.PURGE.<account>
+```
+
+where `<account>` is the unique name of the account.
+
+For example, given the following server config with inline [accounts][accounts]:
+
+```
+accounts: {
+  \$SYS: {
+    users: [{ user: sys, password: sys }]
+  }
+
+  APP: {
+    jetstream: true
+    users: [{ user: app, password: app }]
+  }
+}
+```
+
+Using the CLI, a raw request could be invoked as such:
+
+```
+nats --user sys --password sys \
+  req '$JS.API.ACCOUNT.PURGE.APP' ''
+```
+
+If using [decentralized auth][jwt], the account would be the public key.
+
+```
+nats --user sys --password sys \
+  req '$JS.API.ACCOUNT.PURGE.AAWFSQ7EDAMTCZ2EUZ7B3YPVOF7VFL6ZKUM7IEEMW2IOFXUGTCHMNZEP' ''
+```
+
+[accounts]: https://docs.nats.io/running-a-nats-service/configuration/securing_nats/accounts
+[jwt]: https://docs.nats.io/running-a-nats-service/configuration/securing_nats/auth_intro/jwt
+
+#### New subject mapping functions
+
+NATS supports this concept of [subject mapping][subject-mapping] which, in essence, is a transform of an input subject to an output subject. This can be used for simple remapping, determinstic partitioning, canary deployments, A/B testing, etc. Subject mapping also is used when defining account imports and exports as well as the new republish capability noted above.
```

Comment:
```suggestion
NATS supports this concept of [subject mapping][subject-mapping] which, in essence, is a transform of an input subject to an output subject. This can be used for simple remapping, deterministic partitioning, canary deployments, A/B testing, etc. Subject mapping also is used when defining account imports and exports as well as the new republish capability noted above.
```